docs/content/development/extensions-contrib/graphite.md

```diff
@@ -24,7 +24,8 @@ All the configuration parameters for graphite emitter are under `druid.emitter.g
 |`druid.emitter.graphite.flushPeriod` | Queue flushing period in milliseconds. |no|1 minute|
 |`druid.emitter.graphite.maxQueueSize`| Maximum size of the queue used to buffer events. |no|`MAX_INT`|
 |`druid.emitter.graphite.alertEmitters`| List of emitters where alerts will be forwarded to. |no| empty list (no forwarding)|
- 
+|`druid.emitter.graphite.emitWaitTime` | wait time in milliseconds to try to send the event otherwise emitter will throwing event. |no|1000 (1 sec)|
```

Comment:
IMO the default should be 0, it's rare that people would want to block their query threads waiting for metrics emission rather than just drop some metrics and have queries return faster.